.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please remove this or add to `.gitignore`

Reply:
Working on this for better UI.